<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OuterForm</class>
 <widget class="QWidget" name="OuterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Basic Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/main.ico</normaloff>:/main.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #000000;
}

QToolButton {
	background-color: #000000;
	border: 0px;
}

QLabel {
	color: #FFFFFF;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="../qrc/basicviewer.qrc">:/images/logo.PNG</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string>color: #838FC7;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="btnMinimize">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="text">
           <string>_</string>
          </property>
          <property name="icon">
           <iconset resource="../qrc/basicviewer.qrc">
            <normaloff>:/images/minimize.PNG</normaloff>:/images/minimize.PNG</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnMaximize">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="text">
           <string>#</string>
          </property>
          <property name="icon">
           <iconset resource="../qrc/basicviewer.qrc">
            <normaloff>:/images/maximize.PNG</normaloff>:/images/maximize.PNG</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnClose">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="text">
           <string>x</string>
          </property>
          <property name="icon">
           <iconset resource="../qrc/basicviewer.qrc">
            <normaloff>:/images/close.PNG</normaloff>:/images/close.PNG</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #3F414E;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="MainForm" name="childWidget" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainForm</class>
   <extends>QWidget</extends>
   <header>MainForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
  <include location="../qrc/basicviewer.qrc"/>
 </resources>
 <connections/>
</ui>
